index.html

```diff
@@ -15,7 +15,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Please add a link to the corresponding line of code in the Pull Request If where issues are found and changes are made.